semconv/v1.4.0/http_test.go

```diff
@@ -585,7 +585,7 @@ func TestHTTPServerAttributesFromHTTPRequest(t *testing.T) {
 			},
 			header: http.Header{
 				"User-Agent":      []string{"foodownloader"},
-				"X-Forwarded-For": []string{"1.2.3.4"},
+				"X-Forwarded-For": []string{"203.0.113.195, 70.41.3.18, 150.172.238.178"},
```

Comment:
Can we add this as an additional test case? That way we ensure the original behavior, for a single address, is preserved.

Reply:
That path is covered with the other test cases, _eg._ the one below. Would you still like the additional test case?